config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
+# command to execute in running container
+# passed as argument to exec
+fndmnt = findmnt -n %(tmpfs_path)s 
```

Comment:
AFAICT it should be safe to add `-o FSTYPE`; this makes it easy to test that it's actually the expected `tmpfs` type. (Your test currently verifies only that /run is mounted).